text/0002-grin-governance.md

```diff
@@ -183,7 +183,6 @@ Far too many to all be listed here, but here are some:
 **Rust's governance process**
 
 - [https://rust-lang.github.io/rfcs/1068-rust-governance.html](https://rust-lang.github.io/rfcs/1068-rust-governance.html)
-- [https://predictablynoisy.com/rust-governance](https://predictablynoisy.com/rust-governance)
```

Comment:
@raballew should we add the wayback machine version to retain the information? https://web.archive.org/web/2019*/https://predictablynoisy.com/rust-governance

Reply:
@phyro Not sure but the Rust governance is also described in the link above.